addons/hr_attendance/views/hr_attendance_view.xml

```diff
@@ -27,6 +27,8 @@
                 <field name="out_longitude" string="Longitude (Out)" optional="hidden"/>
                 <field name="in_city" string="City (In)" optional="hidden"/>
                 <field name="out_city" string="City (Out)" optional="hidden"/>
+                <field name="in_location" string="Precise location (In)" optional="hidden"/>
```

Comment:
You should define the string on the field itself otherwise I think this it will generate 2 i18n entries ("In Location" and "Precise location (In)")